hippod/report_generator.py

```diff
@@ -83,70 +83,108 @@ def store_data(app, data, sub_dir):
+                    description += '**Categories**            {}\n'.format(categories)
+                    description += '**Submitter**             {}\n'.format(submitter)
+                    description += '**Responsible**           {}\n'.format(responsible)
+                    description += '**Test-Date**             {}\n'.format(test_date)
+                    description += '-----------------------   ----------\n\n'
+                    for data in achievement_data:
+                        description += '![Description]({})\n'.format(data)
+                    description += str(description_only)
                     text.write(description)
                 return description_path
             else:
-                with open(achievement_path, 'r') as achievement:
-                    content = json.load(achievement)
-                    result = content['result']
+                result = achievement_content['result']
+                submitter = achievement_content['submitter']
+                test_date = achievement_content['test-date']
+                categories = attach_content['categories']
+                responsible = attach_content['responsible']
+
                 with open(description_path, 'r') as text:
-                    description = text.read()
+                    description_only = text.read()
                 with open(description_path, 'w') as text:
-                    description = '''
-# {} #
-
------------------------------------------------------------------------------------
-**Test Result:**    {}
-
------------------------------------------------------------------------------------
-
-                    '''.format(title, result) +\
-                    '\n' + str(description)
+                    description  = '# {} #\n\n'.format(title)
+                    description += '-----------------------   ----------\n'
+                    description += '**Test Result**           {}\n'.format(result)
+                    description += '**Categories**            {}\n'.format(categories)
+                    description += '**Submitter**             {}\n'.format(submitter)
+                    description += '**Responsible**           {}\n'.format(responsible)
+                    description += '**Test-Date**             {}\n'.format(test_date)
+                    description += '-----------------------   ----------\n\n'
+                    for data in achievement_data:
+                        description += '![Description]({})\n'.format(data)
+                    description += str(description_only)
                     text.write(description)
-                    return description_path
+                return description_path
 
 
-        def sanitize_description(description_path):
+        def sanitize_description(self, description_path):
             with open(description_path, 'r') as input_text:
                 in_descr = input_text.readlines()
```

Comment:
also good: lines (plural) later you can do something like

```
for line in lines:
    pass
```

or  `desc_lines`